ykrt/src/mt.rs

```diff
@@ -139,9 +169,78 @@ impl MT {
         }
     }
 
-    /// `Location`s represent a statemachine: this function transitions to the next state (which
-    /// may be the same as the previous state!). If this results in a compiled trace, it returns
-    /// `Some(pointer_to_trace_function)`.
+    pub fn set_loc_code_ptr(loc: *mut Location, code: *const c_void) {
```

Comment:
What does this do? Might benefit from a doc comment.

Reply:
I'll add a comment.

It's needed because when the control point calls `__yktrace_irtrace_compile()` a pointer to JIT code is returned, and we need some way to store that (eventually into the location, but for now) into the global variable that stores the trace pointer.

This isn't very optimal, but I wanted to get something simple working first.

Reply:
I interpret Lukas' silence as, yes, maybe?

Reply:
Can you point out which commit added a comment?

Reply:
It has been addressed!

Reply:
Appears I lied! Sorry. Try 63a9f67.